playbooks/maas-verify.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
once we have maas in the /opt/rpc-maas location there's no need to wrap this the verify playbook here.

Reply:
Cool, works for me.